docs/lakehouse-iceberg-catalog.md

```diff
@@ -36,12 +36,12 @@ Builds with Apache Iceberg `1.5.2`. The Apache Iceberg table format version is `
-| `catalog-backend-name`                             | The catalog name passed to underlying Iceberg catalog backend. Catalog name in JDBC backend is used to isolate namespace and tables.                                                                                                             | Gravitino catalog name | No                                                          | 0.5.2         |
+| Property name                                      | Description                                                                                                                                                                                                                                      | Default value                     | Required                                                    | Since Version |
+|----------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------|-------------------------------------------------------------|---------------|
+| `catalog-backend`                                  | Catalog backend of Gravitino Iceberg catalog. Supports `hive` or `jdbc` or `rest`.                                                                                                                                                               | (none)                            | Yes                                                         | 0.2.0         |
+| `uri`                                              | The URI configuration of the Iceberg catalog. `thrift://127.0.0.1:9083` or `jdbc:postgresql://127.0.0.1:5432/db_name` or `jdbc:mysql://127.0.0.1:3306/metastore_db` or `http://127.0.0.1:9001`.                                                  | (none)                            | Yes                                                         | 0.2.0         |
+| `warehouse`                                        | Warehouse directory of catalog. `file:///user/hive/warehouse-hive/` for local fs or `hdfs://namespace/hdfs/path` for HDFS.                                                                                                                       | (none)                            | Yes                                                         | 0.2.0         |
+| `catalog-backend-name`                             | The catalog name passed to underlying Iceberg catalog backend. Catalog name in JDBC backend is used to isolate namespace and tables.                                                                                                             | `catalog-backend` property value. | No                                                          | 0.5.2         |
```

Comment:
The default value is "**catalog-backend**"  or  "**`catalog-backend` property value**".

Reply:
The property value of `catalog-backend`, and add an example

Reply:
Is it better to use only values like `catalog-backend`?

Reply:
seems a little odd